docs/checkout-extension-post-purchase.md

```diff
@@ -47,65 +57,78 @@ SHOPIFY_API_SECRET=abcdefg
+
+From the `checkout-extension` folder, run the following command:
 
 ```bash
 npm install
 ```
 
-### Update the `embeddedAppHost` variable
+### 4. Update `embeddedAppHost` variable
+
+_The extension needs to know where the main application is being hosted, this is managed via a variable._
 
-In the `src/index.js` file the variable `embeddedAppHost` is currently hardcoded. This needs to be updated to reflect the URL that `ngrok` generated.
+Open the [`src/index.js`](/checkout-extension/src/index.js) file and update the variable `embeddedAppHost` to reflect the URL that `ngrok` generated. It should look like the following:
 
-## Register the extension
+```javascript
+const embeddedAppHost = "https://3db4-76-68-94-203.ngrok.io";
+```
+
+## 5. Register the extension
+
+_Registering your app extension associates it with an app in your Partner organization. This step should be done only once for each app extension._
 
-Register the extension by running the Shopify CLI command
+To register the extension, run the following command:
 
 ```bash
 shopify extension register
 ```
 
-## Push the extension
+## 6. Push the extension
 
-_This will build and compile a production version and push it to the Shopify CDN. (You should see that a build folder has been created with your minified code inside)._
+_Pushing the extension entails of building and compiling a production version of the extension, which is then pushed to the Shopify CDN. (You should see that a build folder has been created with your minified code inside)._
+
+To push the extension, run the following command:
 
 ```bash
 shopify extension push
 ```
 
-You should now see the extension in your app page in your Partner Dashboard under _Extensions_ > _<Your Checkout Name>_.
+## 7. Publish the extension
 
-## Publish the extension
+_When you're ready to release your extension you can publish it._
 
-Once your Theme App Extension code is ready for public use, push the latest code, head to _Extensions > Online Store > Post Check App Extensions_ under your App, click on the extension name, then click _Create Version_ button and once new version appears in your versions list simply click _Publish_ next to it.
+The publish the extension navigate to the following:
 
-## Enable the Post Checkout extension
+_***Partner Dashboard*** > Apps > ***Your App Name*** > Extensions > ***Your Checkout Extension Name***._
 
-One final step is to enable the checkout extension from the Store's admin settings. To do that, go to the _Store Admin Settings_ > _Checkout_ and select the extension you created.
+Once on the extension page, click on the **_Create version_** button and when the new version appears in the version list, click **_Publish_** next to it.
 
-![Checkout menu](images/pc-menu.png)
+> :exclamation: To save time, you can preview a draft of your app extension in a development store before creating a version by selecting the **_DEVELOPMENT STORE PREVIEW_** option from the extension's settings page.
```

Comment:
Maybe add a note here that merchants wont be able to use this unless its published. Development preview should only be used while in active development.

Reply:
added a note